src/main/java/gregtech/common/items/behaviors/ClipboardBehavior.java

```diff
@@ -31,28 +31,30 @@
 
     @Override
     public ModularUI createUI(PlayerInventoryHolder holder, EntityPlayer entityPlayer) {
-        initNBT(holder.getCurrentItem());
         ModularUI.Builder builder = ModularUI.builder(GuiTextures.BACKGROUND, 170, 238);
+        initNBT(holder.getCurrentItem());
```

Comment:
Any reason behind this change?

Reply:
Oh! Originally, in earlier testing, I put a statement to just return the builder on the server side and extract side effects from the second method, but now it no longer needs to be in this order.

Reply:
No problem, was just making sure there wasn't something I was missing